frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/emoji_popover.dart

```diff
@@ -72,18 +73,16 @@ class _EmojiPopoverState extends State<EmojiPopover> {
     return FlowyButton(
       onTap: () => widget.removeIcon(),
       useIntrinsicWidth: true,
-      hoverColor: Theme.of(context).colorScheme.onPrimary,
       text: Row(
         mainAxisSize: MainAxisSize.min,
         mainAxisAlignment: MainAxisAlignment.end,
         children: [
-          svgWidget("editor/delete"),
+          const FlowySvg(name: 'editor/delete'),
```

Comment:
+1 to a `SVGRegistry` so that we don't have these hardcoded strings everywhere. If the svg is renamed, we will have some serious problems. Do you want me to open an issue for that?

Reply:
Yes, Thanks.